### PR TITLE
CI: pin xgo and go to v1.18.7

### PR DESCRIPTION
**Pull request by 6543**

### CODE REVIEW

File: .woodpecker/docker.yml [hunk 1]
```diff
@@ -3,10 +3,10 @@ depends_on:
   - web
 
 variables:
-  - &golang_image 'golang:1.18'
+  - &golang_image 'golang:1.19.3'
```
Comment (lafriks) [inline .woodpecker/docker.yml:6]: Why not `1.19`?

File: .woodpecker/docker.yml [hunk 2]
```diff
@@ -3,10 +3,10 @@ depends_on:
...
   - &node_image 'node:18-alpine'
-  - &xgo_image 'techknowlogick/xgo:go-1.18.x'
-  - &xgo_version 'go-1.18.x'
+  - &xgo_image 'techknowlogick/xgo:go-1.19.3'
+  - &xgo_version 'go-1.19.3'
```
Comment (lafriks) [inline .woodpecker/docker.yml:9]: Same here

File: .woodpecker/docker.yml (hunk 1 above)
Comment (6543) [reply .woodpecker/docker.yml:6]: more pinning ... if it goes wrong as with 1.18 again ...

Comment (6543) [reply .woodpecker/docker.yml:6]: ... well it's an experiment for now if it works or if I need to dig into xgo :(

**Comment by 6543** [comment]
**so the issue is a xgo one and it happened in transition from  v1.18.7 to v1.18.8**

**Review by anbraten — APPROVED** [review]

**Comment by 6543** [comment]
we do need to find the underlying issue long term ...

**Pull request merged**